src/page-viewer/styles.css

```diff
@@ -0,0 +1,6 @@
+.root {
+    float: right;
+    padding: 4px 4px 4px 12px;
```

Comment:
I feel the float and padding styles should not be defined in this component; they specify how to place the component in its context, not how the component itself looks.

Not sure what is a good practice for splitting this up, I guess CSS was not designed with modularity in mind. Maybe we should keep this component purely technical, i.e. not do any styling in here. The overview can just pass `className` down as a prop.